lib/spotify_sdk_web.dart

```diff
@@ -197,7 +184,9 @@ class SpotifySdkPlugin {
       case MethodNames.getAuthenticationToken:
         return await _authorizeSpotify(
             clientId: call.arguments[ParamNames.clientId] as String,
-            redirectUrl: call.arguments[ParamNames.redirectUrl] as String);
+            redirectUrl: call.arguments[ParamNames.redirectUrl] as String,
+            scopes:
+                call.arguments[ParamNames.scope] as String? ?? DEFAULT_SCOPE);
```

Comment:
I think that also works
```suggestion
                call.arguments[ParamNames.scope] as String ?? DEFAULT_SCOPE);
```